### PR TITLE
Pages Objects in test:headless

### PR DESCRIPTION
**Pull request by alejajessi**

### CODE REVIEW

File: src/page/payment-step.page.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class PaymentStepPage {
+  private checkOutButton: ElementFinder;
+
+  constructor () {
+    this.checkOutButton = $('#HOOK_PAYMENT > div:nth-child(1) > div > p > a');
+  }
+
+  public async checkOuttoFiniishPay(): Promise<void> {
```
Comment (leonleo997) [inline src/page/payment-step.page.ts]: typo on `finish`

File: src/page/product-list.page.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { $, ElementFinder } from 'protractor';
+
+export class ProductListPage {
+  private selectTShirt: ElementFinder;
+
+  constructor () {
+    this.selectTShirt = $('#center_column a.button.ajax_add_to_cart_button.btn.btn-default');
+  }
+
+  public async selectClothes_TShirt(): Promise<void> {
```
Comment (leonleo997) [inline src/page/product-list.page.ts]: use camelCase

File: src/page/order-summary.page.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export class OrderSummaryPage {
+  private finalButton: ElementFinder;
+
+  constructor () {
+    this.finalButton = $('#center_column > div > p > strong');
+  }
+
+  public async summaryOrder(text: string): Promise<void> {
+    await expect(this.finalButton.getText()).toBe(text);
```
Comment (leonleo997) [inline src/page/order-summary.page.ts]: you should not add assertions into a PO

File: test/buy-tshirt.spec.ts [hunk 4]
```diff
@@ -1,41 +1,59 @@
...
 
-    await $('#cgv').click();
+    await shippingStepPage.confirmSpace();
     await(browser.sleep(3000));
 
-    await $('#form > p > button > span').click();
+    await shippingStepPage.continueToPay();
     await(browser.sleep(3000));
-    await $('#HOOK_PAYMENT > div:nth-child(1) > div > p > a').click();
+    await paymentStepPage.checkOuttoFiniishPay();
```
Comment (leonleo997) [inline test/buy-tshirt.spec.ts]: typo on `finish`

File: test/buy-tshirt.spec.ts [hunk 5]
```diff
@@ -1,41 +1,59 @@
...
     await(browser.sleep(3000));
-    await $('#cart_navigation > button > span').click();
+    await bankPaymentPage.checkOuttoFinish();
     await(browser.sleep(3000));
 
-    await expect($('#center_column > div > p > strong').getText())
-      .toBe('Your order on My Store is complete.');
+    await orderSummaryPage.summaryOrder(textConfirmation);
```
Comment (leonleo997) [inline test/buy-tshirt.spec.ts]: the assertion should be into the spec file, not in the PO file

File: src/page/order-summary.page.ts (hunk 3 above)
Comment (alejajessi) [reply src/page/order-summary.page.ts]: ¿Expect? then I should put in buy-tshirt example: await expect( orderSummaryPage.summaryOrder() ....etc ?

File: src/page/product-list.page.ts (hunk 2 above)
Comment (alejajessi) [reply src/page/product-list.page.ts]: I don't understand bff

**Pull request merged**